src/jit/lsra.cpp

```diff
@@ -8536,6 +8545,13 @@ void LinearScan::recordMaxSpill()
         maxSpill[TYP_FLOAT] += 1;
     }
 #endif // _TARGET_X86_
+
+    if (compiler->opts.compUseSoftFP)
+    {
+        JITDUMP("Adding a spill temp for moving target address to a register.\n");
+        maxSpill[TYP_INT] += 1;
```

Comment:
Of course, this cannot be a solution since it will allocate one more temps for all methods.